### DISCUSSION TITLE
Feature: integration with Effector?

### DISCUSSION THREAD

**Discussion by elite174** [solidjs/solid#741] [Q&A] [answered]
This is a question actually. Are there plans to have something like `solid-effector` lib?

**Comment by ryansolid** [comment, marked as answer]
I don't believe I've seen a port of that one, but there has been some work in the community on state management(store-on, Zustand, etc..). Many people just use Solid's built-in reactive state management since it works independently of components already. But nothing to stop someone from implementing this.

From a core perspective we are focused core mechanics and already provide the basic tools for state management and these sort of integrations. So I wouldn't expect an official integration but I encourage anyone from the community to pick this up if they find value in it.

**Reply by elite174** [reply]
Thank you for the reply!
